website/src/theme/MDXComponents/InlineCode.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import Link from '@docusaurus/Link';
+
+const MarkdownInlineCodePrefix = 'md ';
+
+export default function InlineCode(props) {
+  if (
+    typeof props.children === 'string' &&
+    props.children.startsWith(MarkdownInlineCodePrefix)
+  ) {
+    return (
+      <MarkdownInlineCode
+        {...props}
+        children={props.children.slice(MarkdownInlineCodePrefix.length)}
+      />
+    );
+  }
+  return <code {...props} />;
+}
+
+// An inline code block with some basic Markdown support
+// Currently only supports links, which should be good enough...
+// To enable Markdown support, use the "`md " prefix
+// Syntax example: `md ({nativeEvent: [PressEvent](pressevent)})`
+const MarkdownInlineCode = React.memo(function MarkdownInlineCodeInner(props) {
+  const children = linkify(props.children);
+  return <code {...props} children={children} />;
+});
+
+// Gives the ability to use basic Markdown links inside inline code blocks
+// We use RegExp because a full Markdown parser would be quite heavy
+// See https://github.com/facebook/react-native-website/pull/3807
+function linkify(input) {
```

Comment:
@cipolleschi @Simek @cortinico 

This is the code that enables the ability to use Markdown links inside inline code blocks, leading to this outcome:

![CleanShot 2023-08-04 at 17 53 43](https://github.com/facebook/react-native-website/assets/749374/e5a595af-2fd9-4944-86b2-f7c208ed75e8)

This code is included in the client bundles, and I didn't want to include a heavy Markdown parser just for that use case. For that reason I choose to implement partial support (just markdown links), use regexp (less robust) and keep this feature opt-in.